addOns/exim/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Added
+- Pcap/ng import GUI (Issue 4812, WIP)
```

Comment:
I'd suggest just making this generic to cover all the initial work.
`- Initial PCAP import support (Issue 4812).`
or something like that.